src/Tickets/Commerce/Gateways/Stripe/Signup.php

```diff
@@ -59,11 +64,26 @@ public function generate_signup_url() {
 	 */
 	public function generate_disconnect_url() {
 
+		$webhooks = tribe( Webhooks::class );
+
+		$known_webhooks = tribe_get_option( $webhooks::OPTION_KNOWN_WEBHOOKS, [] );
+
+		$current_signing_key = tribe_get_option( $webhooks::$option_webhooks_signing_key );
+
+		$known_webhooks = array_filter(
+			$known_webhooks,
+			function ( $signing_key ) use ( $current_signing_key ) {
+				return $signing_key === $current_signing_key;
+			}
+		);
```

Comment:
There should be only one, right?
Why not use something like 
```php
$invert = array_flip( $known_webhooks );
$known_webhooks = array_flip( $invert[$current_signing_key] ?? [] );
```

Reply:
Its not exacactly what i was looking for but used your array_flip to idea.

$known_webhooks should store up to 3 key => pairs (webhook_id => webhook_secret), but for disable we only want to send the current webhook being used and that only if we know its id.